recipes/svdb/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.7" %}
+{% set md5sum = "2764c03866f3aa827bec3fcc50fa1742" %}
+
+package:
+  name: svdb
+  version: "{{ version }}"
+
+source:
+  fn: svdb-{{ version }}.tar.gz
+  url: https://github.com/J35P312/SVDB/archive/SVDB-{{ version }}.tar.gz
+  md5: "{{ md5sum }}"
```

Comment:
It's [usually preferred to use `sha256`](http://bioconda.github.io/guidelines.html#hashes).